executor/write_test.go

```diff
@@ -805,15 +806,48 @@ func (s *testSuite) TestLoadDataEscape(c *C) {
 	checkCases(tests, ld, c, tk, ctx, selectSQL, deleteSQL)
 }
 
-func makeLoadDataInfo(column int, ctx context.Context, c *C) (ld *executor.LoadDataInfo) {
+// reuse TestLoadDataEscape's test case :-)
+func (s *testSuite) TestLoadDataSpecifiedCoumns(c *C) {
+	defer func() {
+		s.cleanEnv(c)
+		testleak.AfterTest(c)()
+	}()
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test; drop table if exists load_data_test;")
+	tk.MustExec(`create table load_data_test (id int PRIMARY KEY AUTO_INCREMENT, c1 int, c2 varchar(255) default "def", c3 int default 0);`)
+	tk.MustExec("load data local infile '/tmp/nonexistence.csv' into table load_data_test")
```

Comment:
Use `load data local infile '/tmp/nonexistence.csv' into table load_data_test (c1,c2) ` instead of it or remove it .